web/config/sync/filter.format.dbcdk_community_profiles__description.yml

```diff
@@ -0,0 +1,23 @@
+uuid: bda9e83c-d51b-40f6-a306-6cd8b0496666
+langcode: en
+status: true
+dependencies: {  }
+name: 'DBCDK Community Profiles - Description'
+format: dbcdk_community_profiles__description
```

Comment:
Does it make sense to create a specific format for each field as indicated by this name? Could we provide just one for rich content from the service?